doc/source/reference/freud.bib

```diff
@@ -243,6 +243,17 @@ @article{Fransson2021
   url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/adts.202000240}
 }
 
+@article{Vymetal2011,
+  author = {Vym{\v{e}}tal, Jiř{\'{i}} and Vondr{\'{a}}{\v{s}}ek, Jiř{\'{i}}},
+  title = {Gyration- and inertia-tensor-based collective coordinates for metadynamics. Application on the conformational behavior of polyalanine peptides and Trp-cage folding},
+  journal = {Journal of Physical Chemistry A},
+  year = {2011},
+  volume = {115},
+  number = {41},
+  pages = {11455--11465},
+  doi = {https://doi.org/10.1021/jp2065612},
+  url = {https://pubs.acs.org/doi/full/10.1021/jp2065612},
+
```

Comment:
The documentation is failing to build because of a missing closing parenthesis in this citation.

Reply:
Thanks, totally forgot to check the doc build after resolving the merge.